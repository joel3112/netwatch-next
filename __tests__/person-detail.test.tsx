import { render } from '@testing-library/react';
import PersonDetailPage from '@/pages/person/[personId]';

describe('Tests person detail page', () => {
  test('renders page correctly', () => {
    const { container } = render(<PersonDetailPage />);

    expect(container).toMatchSnapshot();
  });
});
