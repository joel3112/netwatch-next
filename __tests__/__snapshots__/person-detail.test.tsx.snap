// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests person detail page renders page correctly 1`] = `
<div>
  <div
    class="wrapper hasMargins"
  />
</div>
`;
